srcpkgs/python3-pmw/template

```diff
@@ -0,0 +1,19 @@
+# Template for python-pmw
+pkgname=python3-pmw
+version=2.0.1
+revision=1
+noarch=yes
+short_desc="Python 3 Tkinter widget toolkit"
+homepage="http://pmw.sourceforge.net/"
+license="MIT"
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+wrksrc=Pmw-${version}
+build_style=python3-module
+hostmakedepends="python3-devel tk-devel"
+depends="tk python3"
+distfiles="${PYPI_SITE}/P/Pmw/Pmw-${version}.tar.gz"
+checksum=0b9d28f52755a7a081b44591c3dd912054f896e56c9a627db4dd228306ad1120
+
+post_install() {
+	vlicense ${FILESDIR}/LICENSE.txt
+}
```

Comment:
Does [L20](https://github.com/void-linux/void-packages/pull/1473/files#diff-95eabced7a2601a3c584faed947d0274R20) fix this? I ask because GitHub doesn't seem to be registering a change there. I've edited the template in Vim and pressed return at the end of this line, saved it and git hasn't registered a change, so hence I'm guessing [L20](https://github.com/void-linux/void-packages/pull/1473/files#diff-95eabced7a2601a3c584faed947d0274R20) is what you wanted.

Reply:
@pullmoll Guessing you've forgotten about this or weren't notified. Does L20 fix this?

Reply:
You know what? I'm going to merge this and modify it then you can see what we were talking about.
Sorry, I was busy with gcc-8.2.0 and dependencies.